### PR TITLE
gst: update version 3.1->3.2.5

### PR DESCRIPTION
**Pull request by MarcusCalhoun-Lopez**
Fixes https://trac.macports.org/ticket/27730

###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.13
Xcode 9.4

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [X] checked your Portfile with `port lint`?
- [X] tried existing tests with `sudo port test`?
- [X] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: lang/gst/Portfile [hunk 1]
```diff
@@ -3,8 +3,8 @@
 PortSystem              1.0
 
 name                    gst
-version                 3.1
-revision                7
+version                 3.2.5
+#revision                0
```
Comment (pmetzger) [inline lang/gst/Portfile]: I would just nuke the revision line rather than commenting it out. That is our practice in these cases after all.

Comment (MarcusCalhoun-Lopez) [reply lang/gst/Portfile]: Change made.

**Comment by pmetzger** [comment]
LGTM.

**Comment by pmetzger** [comment]
Merged. Thanks, @MarcusCalhoun-Lopez!

**Pull request merged**